src/mcnp/mcnp_funcs.cpp

```diff
@@ -222,13 +223,13 @@ void write_cell_cards(std::ostringstream& lcadfile, const char* mcnp_version_maj
       std::string mat_name = DMD->volume_material_property_data_eh[entity];
       // if we not vacuum or graveyard
       if (mat_name.find(vacuum_str) == std::string::npos && mat_name.find(graveyard_str) == std::string::npos) {
-        if (workflow_data->material_library.count(mat_name) == 0) {
+        pyne::Material material = workflow_data->material_library.get_material(mat_name);
```

Comment:
I am not a fan of this approach.  See my comment in `UWUW` about defining `workflow->material_library` as a `pyne::mat_map`....